packages/api/src/schema/api-schema.yaml

```
@@ -1845,13 +1913,105 @@ components:
           description: The error rate for the asset
         existsBeforeStart:
           type: number
-          description:
+          description: >-
             The percentage of sessions that existed before the asset started
             playing
+    create-asset-url:
+      type: object
+      required:
+        - url
+        - name
+      properties:
+        url:
+          type: string
+          example: $EXTERNAL_URL
+        name:
+          type: string
+          example: Example name
+    update-record-status:
+      type: object
+      required:
+        - record
+      properties:
+        record:
+          type: boolean
+          example: true
+    playback-info:
+      type: object
+      properties:
+        type:
+          type: string
+          example: vod
+        meta:
+          type: object
+          properties:
+            source:
+              type: array
+              items:
+                type: object
+                properties:
+                  hrn:
+                    type: string
+                    example: MP4
+                  type:
+                    type: string
+                    example: html5/video/mp4
+                  url:
+                    type: string
+                    example: >-
+                      https://asset-cdn.lp-playback.monster/hls/1bde4o2i6xycudoy/static360p0.mp4
+                  size:
+                    type: number
+                    example: 494778
+                  width:
+                    type: number
+                    example: 204
+                  height:
+                    type: number
+                    example: 360
+                  bitrate:
+                    type: number
+                    example: 449890
+      required:
+        - type
+        - meta
 paths:
   /stream:
     post:
-      description: Creates a stream
+      tags:
+        - Stream
+      summary: Create a stream
+      description: >
+        POST /stream creates a stream object. The configuration on this parent
+        stream object is inherited by all future child session objects.
+
+
+        It is important to note, that the stream object is the core building
+        block of the Livepeer Studio platform. In Livepeer Studio, the stream
+        object has unique configuration data and metadata about all livestream
+        sessions that are associated with the stream.
+
+
+        Historically, a stream object with a parentId represented a single
+        livestream session. This type of "child-session" stream object will be
+        deprecated in an upcoming version of the API in favor of the session
+        object.
```

This "child session" situation has finally already changed with recording v2! We should remove those remarks.
```suggestion
        POST /stream creates a stream object. The configuration on this stream object
        is inherited by all live sessions using this stream's streamKey.


        It is important to note, that the stream object is the core building
        block of the Livepeer Studio platform. In Livepeer Studio, the stream
        object has unique configuration data and metadata about all livestream
        sessions that are associated with the stream.
```